js/README.md

```diff
@@ -32,626 +32,66 @@
-├── components/
-│   ├── alert.css
-│   ├── button.css
-│   └── grid.css
-├── utils/
-│   ├── display.css
-│   ├── position.css
-│   └── text.css
-├── vendor/
-│   └── ...
-└── main.css
-</pre>
+    eslint: [`prefer-const`](https://eslint.org/docs/rules/prefer-const.html), [`no-const-assign`](https://eslint.org/docs/rules/no-const-assign.html)
 
-### Base Styles
-
-The `base/` directory should contain foundational styles and other global dependencies. There should be few (if any) class definitions within this directory. In addition to common variables, this would also be an appropriate place to put mixins or functions if a preprocessor is in use.
-
-- Variables, mixins and functions should be organized in their own separate files.
-
-    ```css
-    /* base/variables.css */
-
-    :root {
-      --font-family-sans: popular-sans, "Helvetica Neue", sans-serif;
-      --font-size-base: 22px;
-      --transition-duration: 360ms;
-    }
-
-    @custom-media --width-min-md screen and (min-width: 30em);
-    @custom-media --width-min-lg screen and (min-width: 60em);
-    ```
-
-- Each file should only be responsible for providing base styles for a designated group of related elements.
-
-    ```css
-    /* base/scaffolding.css */
-
-    * {/*...*/}
-
-    html {/*...*/}
-
-    body {/*...*/}
-    ```
-
-    ```css
-    /* base/forms.css */
-
-    label {/*...*/}
-
-    input {/*...*/}
-
-    button {/*...*/}
-    ```
-
-    ```css
-    /* base/typography.css */
-
-    h1,
-    h2,
-    h3 {/*...*/}
-
-    pre,
-    code {/*...*/}
-    ```
-
-### Components
-
-The `components/` directory should have the largest number of files, because most of the CSS should be written as small, reusable components.
-
-- Each file should contain only one component and, if possible, only class selectors with a common namespace for that component:
-
-    ```css
-    /* components/alert.css */
-
-    .Alert {/*...*/}
-
-    .Alert--fixed {/*...*/}
-
-    .Alert-header {/*...*/}
-
-    .Alert-closeButton {/*...*/}
+    Nope.
+    ```js
+    var a = 1;
+    var b = 2;
     ```
-
-    ```css
-    /* components/button.css */
-
-    .Button {/*...*/}
-
-    .Button--large {/*...*/}
-
-    .Button--small {/*...*/}
+    Yep!
+    ```js
+    const a = 1;
+    const b = 2;
     ```
 
-### Utilities
-
-The `utilities/` directory is where definitions for utility classes (or "helpers") should occur. Unlike components, there may be many of these class definitions per file. Their organization should be according to what kind of properties they affect. See [SUIT CSS Utils] for a nice example of this organization.
-
-- Each file should contain only classes prefixed with `u-` to identity them as utilities.
+  <a name="references--disallow-var"></a><a name="1.2"></a>
+  - [1.2](#references--disallow-var) If you must reassign references, use `let` instead of `var`. 
 
-    ```css
-    /* utilities/margin.css */
+    > Why? `let` is block-scoped rather than function-scoped like `var`.
```

Comment:
Which is bad because...?

(I'm not saying this is wrong, just that this is stated so matter-of-factly and I don't understand the "why" of the rule.)